src/connections.jl

```diff
@@ -161,6 +161,19 @@ Returns `L/(1+L)` or `P1/(1+P1*P2)`
 feedback(L::TransferFunction) = L/(1+L)
 feedback(P1::TransferFunction, P2::TransferFunction) = P1/(1+P1*P2)
 
+function feedback(G1::TransferFunction{<:TimeEvolution,<:SisoRational}, G2::TransferFunction{<:TimeEvolution,<:SisoRational})
+    if !issiso(G1) || !issiso(G2)
+        error("MIMO TransferFunction feedback isn't implemented.")
+    end
+    G1num = G1.matrix[1,1].num
```

Comment:
```suggestion
    G1num = numpoly(G1)[]
```
`numpoly/denpoly` should be used instead I guess, in case the internal representation changes

Reply:
Good idea, I'll update. Is `only(x)` or `x[]` recommended, I haven't seen the latter one, but I guess they are equivalent?